modules/local/fusionreport/download/main.nf

```diff
@@ -6,18 +6,17 @@ process FUSIONREPORT_DOWNLOAD {
     container "docker.io/clinicalgenomics/fusion-report:3.1.0"
 
     output:
-    path "fusiongdb2.db"    , emit: fusiongdb2
-    path "mitelman.db"      , emit: mitelman
-    path "cosmic.db"        , emit: cosmic, optional: true
-    path "*.txt"            , emit: timestamp
-    path "*.log"            , emit: log
-    path "versions.yml"     , emit: versions
+    tuple val(meta), path("fusionreport_dbs"), emit: fusionreport_db
+    path "versions.yml"                      , emit: versions
 
     script:
+    meta = [id: 'fusionreport_dbs']
```

Comment:
Shouldn't this be 
`def meta = [id: 'fusionreport_dbs']`

And then
```
output:
tuple env(meta), path("fusionreport_dbs"), emit: fusionreport_db
```

Make it a local variable to avoid problems as a global variable.

Reply:
unfortunately that breaks the process.